Sources/Queues/QueuesCommand.swift

```diff
@@ -4,20 +4,15 @@ import Vapor
 import NIOConcurrencyHelpers
 import NIOCore
 import Atomics
-#if os(Linux)
-import Glibc
-#else
-import Darwin.C
-#endif
 
 /// The command to start the Queue job
-public final class QueuesCommand: Command {
-    /// See `Command.signature`
+public final class QueuesCommand: AsyncCommand, @unchecked Sendable {
```

Comment:
Can we avoid making this unchecked now or is the `Box` causing issues?

Reply:
It's the `Box`, yeah - `DispatchSourceSignal` isn't `Sendable` 🤦‍♀️ But that's workaround-able with a `@preconcurrency import Dispatch`, same thing you did with `ServeComand`.